fix issue with disabled migrate button (#137) (#138)
TheJumpCloud/jumpcloud-ADMU: ModuleChangelog.md

## 2.7.6

Release Date: August 21, 2024

#### RELEASE NOTES

> This fixes an issue with disabled Migrate Button

Reviewer: ```suggestion
This release addresses a UI issue with the ADMU where the "Migrate Profile" button could not be selected even though all the required fields were satisfied.
```

Author: The issue was that Test-Button needed an Org ID for the if statement logic to enable the migrate button. Before this change, Test-Button gets called before initializing the OrgID when API key is inputted. Fix was to move the Test-Button after OrgId is initialized.

Reviewer: Thank you!

#### Bug Fixes:
```

> * Fixed an issue with Migrate Button disabled when there is a username is inputted before JC API key

Reviewer: ```suggestion
* Fixed an issue with ADMU UI "Migrate Profile" button where it remained disabled even though all the required fields were satisfied.
```

```
## 2.7.5

> Release Date: August 21, 2024

Reviewer: ```suggestion
Release Date: August 28, 2024
```

Reviewer: We should be checking this in our tech debt cards
